tests/typing/test_typing.py

```diff
@@ -0,0 +1,127 @@
+# flake8: noqa: E501
+
+import subprocess
+import sys
+
+from pathlib import Path
+from typing import List, Tuple
+
+import pytest
+
+
+CURRENT_DIR = Path(__file__).parent
+
+
+def run_check(path_location: str) -> str:
+    """Use mypy to check the given path location and return the output."""
+
+    mypy_path = "mypy"
+    path = CURRENT_DIR / path_location
+    command = [mypy_path, path.resolve().as_posix()]
+
+    process = subprocess.run(
+        command,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        universal_newlines=True,
+    )
+    output = process.stdout + process.stderr
```

Comment:
Do we need to check if stuff is in stderr too for any specific reason?

Reply:
If there is a typing error, we don't really care. It does not need to cause this test to fail. Here, we only care about the notice of the `reveal_type`. We will let `tox -e type-checking` report on the actual issue. Conceivably, `mypy` could fail, but all these tests pass.